lib/copy.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var hoek = require('hoek');
+
+var ghReq = require('./gh_request.js');
+
+function labelsPath (owner, repo) {
+  return '/repos/' + owner + '/' + repo + '/labels';
+}
+
+module.exports = function copy (request, reply, decoded) {
+  var repoData = request.payload;
+  var userAgent = decoded.agent;
+  var githubToken = decoded.token;
+  var gh = ghReq(githubToken, userAgent);
+  var labelsSet = 0;
+
+  gh({
+    method: 'GET',
+    path: labelsPath(repoData.sourceOwner, repoData.sourceRepo)
+  }, function (err, labels) {
+    hoek.assert(!err, err);
```

Comment:
what _happens_ when there _is_ an error...? are we _handling_ it or displaying it to the client? e.g: https://github.com/dwyl/hapi-error ?